client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/index.tsx

```diff
@@ -300,6 +310,42 @@ const designSetup: Step = function DesignSetup( { navigation, flow } ) {
 		);
 	}
 
+	function previewGeneratedDesign( design: Design ) {
+		const stepContent = (
+			<GeneratedDesignPreview
+				slug={ design.slug }
+				previewUrl={ getDesignPreviewUrl( design, { language: locale } ) }
+			/>
+		);
+
+		return (
+			<StepContainer
+				stepName={ 'design-setup' }
+				stepContent={ stepContent }
+				hideSkip
+				hideNext={ false }
+				className={ classnames( 'design-setup__preview', 'design-picker__is-generated' ) }
+				nextLabelText={ 'Continue' }
+				backLabelText={ 'Pick another' }
+				goBack={ handleBackClick }
+				goNext={ () => pickDesign() }
+				recordTracksEvent={ recordTracksEvent }
+			/>
+		);
+	}
+
+	let stepContent = <div />;
```

Comment:
We might not need this line as we could define `stepContent` at L359 directly

Reply:
You're right, we don't!